tests/tree/test_syntax_tree_builder.py

```diff
@@ -139,14 +140,26 @@ def test_duplicate_multivector_builtin_types_are_not_produced():
         v1 : multivector< 14, S1 >;
     }
 
+enum Enum1 : u16 {
+ A = 1, B=13, C
+}
+
+struct XXX { e : Enum1; f : .ns.Enum1 : 4; }
```

Comment:
`.` in the beginning is optional, right?

Reply:
Actually not. The current implementation requires "." to start at the global namespace... similar to C++'s "::". I think we could change this by adding a few more lookup rules (e.g. look into global if lookup fails in your own namespace)